client/app/scripts/liveblog-security.service.js

```diff
@@ -32,7 +32,7 @@ angular.module('liveblog.security', [])
             return $rootScope.currentUser._id === archive.original_creator || canPublishAPost();
         }
         function canAccessSettings(archive) {
-            return canCreateABlog() && isUserOwnerOrAdmin(archive);
+            return canCreateABlog() && (isUserOwnerOrAdmin(archive) || isMemberOfBlog(archive));
```

Comment:
it must have canCreateABlog right as well and that means admins and editors. Contributors do not have the rights to create blog. Tested the thing. You can check how it works on the branch